### DISCUSSION TITLE
"Cu" is the chemical symbol for which metal?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#79] [Galaxy Trivia] [answered]
**Hint:** 🗽

**Comment by Tynab** [comment, marked as answer]
"Cu" is the chemical symbol for the metal copper. Copper is a ductile and malleable metal with excellent electrical and thermal conductivity. It is a soft, reddish-brown metal that has been used for thousands of years for tools, weapons, and decorative objects. Today, copper is widely used in electrical wiring, plumbing, construction, and various industrial applications. It is also a component of many alloys, such as bronze and brass.
